README.md

```diff
@@ -1,55 +1,51 @@
 [![Build Status](https://travis-ci.org/iqbal-lab-org/gramtools.svg?branch=master)](https://travis-ci.org/iqbal-lab-org/gramtools)
 
 # gramtools
-Genomes evolve by recombination and mutation. 
-gramtools uses a reference graph to  model new genomes as recombinants (mosaics) of previous genomes,
-(by "quasimapping" reads to the graph) and then genotyping within the graph.
+**TL;DR** Genome inference using prior information encoded as a refernce graph.
 
-## Motivation
-gramtools finds the nearest mosaic from a reference panel of genomes to a sample(a "personalised reference"). The user can then either
-* use traditional mapping-based variant calling, 
-* or, wait until we finish implementing variant discovery on top of the graph.
-
-We recommend the first option for now.
+Gramtools builds a directed acyclical graph (DAG) of genetic variance from a population of genomes. Given a sample of reads, the graph is then annotated with quasimap coverage information. A personal reference genome for the population can then be infered from the annotated graph.
```

Comment:
replace variance with variation

Reply:
Also replace "a sample of reads" with "sequence data from a haploid individual (diploid forthcoming)"

Reply:
replace "the graph is then annotated with quasimap coverage information"
with
"alleles on the graph are annotated with coverage information"

Reply:
replace "A personal reference genome for the population" with
"A personalised reference genome for the sample being studied (the closest recombinant of genomes in the graph)"

Reply:
Could also add (as everyone will want to know)
"Future updates will add diploid and mixed ploidy support"

Reply:
Thats fine by me